src/Day01.kt

```diff
@@ -2,23 +2,25 @@
 fun main() {
     fun part1(input: List<String>): Int {
         var checkSum = 0
-        for (values in input) {
-            val num = values.split(' ')
-            checkSum += num.maxOf { it.toInt() } - num.minOf { it.toInt() }
+        for (line in input) {
+            val values = line.split("\t")
```

Comment:
As above, but now that we've swapped `values` for `line` in the `for` we can use `values` here instead of the less descriptive (and somewhat inaccurate) `num`.